src/health-check.php

```diff
@@ -48,6 +48,9 @@
 // Set the minimum cURL version that we've tested that core works with.
 define( 'HEALTH_CHECK_CURL_MIN_VERSION', '7.38' );
 
+// Enqueue accordion script only for this page.
+wp_enqueue_script( 'jquery-ui-accordion' );
```

Comment:
The enqueue here needs to be put in a proper location, most likely `class-healthcheck.php`.

Reply:
Adding it as a dependency for our own JS file is probably the simplest approach.

Reply:
I was thinking to actually enqueue it only on the tools page somehow when I was making it, but I couldn't think of a clear way except just adding it in there but I didn't like it and moved it on the main php. I'm not sure if there's something like is_page() for the admin side to be honest. Maybe a little help here from your mastermind to achieve an even more focused load? 😄